app/views/categories/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <% @categories.each do |category| %>
   <div class="row">
-    <h1><%= category.name %></h1>
+    <h1><%= category.name if category %></h1>
```

Comment:
Yes I did! I seemed to have fixed it by making sure my fabricated videos were set with a category id as well.